exporter/awsemfexporter/datapoint.go

```diff
@@ -196,12 +196,12 @@ func (dps summaryDataPointSlice) At(i int) (dataPoint, bool) {
 	}, retained
 }
 
-// createLabels converts OTel StringMap labels to a map
+// createLabels converts OTel AttributesMap attributes to a map
 // and optionally adds in the OTel instrumentation library name
-func createLabels(labelsMap pdata.StringMap, instrLibName string) map[string]string {
-	labels := make(map[string]string, labelsMap.Len()+1)
-	labelsMap.Range(func(k, v string) bool {
-		labels[k] = v
+func createLabels(attributes pdata.AttributeMap, instrLibName string) map[string]string {
+	labels := make(map[string]string, attributes.Len()+1)
+	attributes.Range(func(k string, v pdata.AttributeValue) bool {
+		labels[k] = v.StringVal()
```

Comment:
Does StringVal work for non string type attrs too?

Reply:
StringVal will return an empty string if the type is not a string. There is an issue to track the work to transform attributes of any type to strings https://github.com/open-telemetry/opentelemetry-collector/issues/3815

Reply:
I am making the changes for emfexporter @codeboten